src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size + $gap) * $col-count - $gap;
```

Comment:
Ensure that the variables $day-size, $gap, $col-count, and $padding are defined elsewhere in your SCSS files. If they are not defined, this will cause a compilation error.